webapp/Connector/src/plugin/DocumentValidation.js

```diff
@@ -73,7 +73,13 @@ Ext.define('Connector.plugin.DocumentValidation', {
         LABKEY.Ajax.request({
             url: LABKEY.ActionURL.buildURL("cds", "validateStudySchemaLink.api"),
             params: {
-                filename: schema_link
+
+                // Support Ticket 40760: Treatment and assay schemas link not showing up on Learn/Study pages
+                // Treatment and Assay schema files reside at '/_webdav/home/files/@files/static/',
+                // however, this link is stored as '/_webdav/home/files/%40files/static/' in cds.study, where '@' is already encoded to %40;
```

Comment:
Sounds like Drienna should fix the study.txt file, instead of this fix

Reply:
Ah ok, thought that those paths were stored in that manner (with %40 instead of @) for some valid reason, but if data can be changed then that would be great! I'll update the ticket requesting that.

Reply:
This change is now reverted, and ticket is updated to request the change in data.